include/swift/SIL/MemAccessUtils.h

```diff
@@ -406,107 +418,53 @@ class AccessedStorage {
   bool operator==(const AccessedStorage &) const = delete;
   bool operator!=(const AccessedStorage &) const = delete;
 };
-} // end namespace swift
+} // namespace swift
 
 namespace llvm {
-
+using swift::AccessedStorage;
```

Comment:
Question. Does this inject swift::AccessedStorage into llvm for anyone who includes this header?